### PR TITLE
A simple component displaying Discourse post

### PR DESCRIPTION
**Pull request by singulart**
Now without any Open API magic ;)

### CODE REVIEW

File: packages/next-frontend/components/discourse.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+const DiscoursePost: React.FC<DiscoursePostProps> = (
+  props: DiscoursePostProps
+) => {
+  const [post, setPost] = useState({} as GetPostResponse);
+
+  useEffect(() => {
+    if (!post.id) {
+      axios
```
Comment (erak) [inline packages/next-frontend/components/discourse.tsx:25]: I don't know what's the general agreement on dependency management, but I used `axios` in the leaderboard PR before switching to [`fetch`](https://github.com/HypeDAO/HypeDAO/pull/8/files#diff-f51529105cea8ba9387091f3e88cce1f6ddf214250a9627dd29e576c9424a115R75). My reasoning was to have less dependencies. @EV3RETH What's your take on this?

**Comment by EV3RETH** [comment]
Thank you for simplifying the code you were using. In addiction to making the deployments pass this also makes our code base simpler and cleaner which is good for all.

File: packages/next-frontend/components/discourse.tsx (hunk 1 above)
Comment (EV3RETH) [reply packages/next-frontend/components/discourse.tsx:25]: I think the less dependancies the better as well

Comment (EV3RETH) [reply packages/next-frontend/components/discourse.tsx:25]: I also like having less dependencies

**Review by EV3RETH — CHANGES_REQUESTED** [review]
This is looking much cleaner, thank you.

a few things I would like done before we get this merged in.
- Remove the word "demo" the page url and title, Something simple like "Posts" will do fine.
- Please add the following posts in this order to the lists and remove the demo ones.
1) https://gov.near.org/t/artist-community-task-events-visions/5308
2) https://gov.near.org/t/artist-introductions/4717
3) https://gov.near.org/t/roadmap-for-hypedao-xyz-featured-artist-pages-synergistic-promotion-functionality-for-hype-and-integrated-mintbase-marketplace/4809

**Comment by singulart** [comment]
> This is looking much cleaner, thank you.
> 
> a few things I would like done before we get this merged in.
> 
> * Remove the word "demo" the page url and title, Something simple like "Posts" will do fine.
> * Please add the following posts in this order to the lists and remove the demo ones.
> 
> 1. https://gov.near.org/t/artist-community-task-events-visions/5308
> 2. https://gov.near.org/t/artist-introductions/4717
> 3. https://gov.near.org/t/roadmap-for-hypedao-xyz-featured-artist-pages-synergistic-promotion-functionality-for-hype-and-integrated-mintbase-marketplace/4809

Done!

**Review by EV3RETH — APPROVED** [review]

**Pull request merged**